pro/src/components/Header/HeaderDropdown/HeaderDropdown.tsx

```diff
@@ -55,6 +55,14 @@ export const HeaderDropdown = () => {
     })) ?? []
   )
 
+  const { pathname } = useLocation()
+  const IN_STRUCTURE_CREATION_FUNNEL = pathname.startsWith(
+    '/parcours-inscription'
+  )
+  const canSeeHisProfile = !(
+    IN_STRUCTURE_CREATION_FUNNEL && offererOptions.length === 0
+  )
```

Comment:
Je suis peut-être fatigué, mais ça serait peut-être plus lisible sans la négation.

J'ai d'ailleurs l'impression que le test n'est pas le bon. Si je ne suis pas sur l'url ET que j'ai déjà une structure, je ne verrais plus le profile ?

```suggestion
  const hideProfile = IN_STRUCTURE_CREATION_FUNNEL && offererOptions.length === 0
```

Reply:
Le test est bon puisque la négation englobe l'intégralité. On ne doit pas voir le profile uniquement si on est dans le funnel et qu'on a pas d'offerer.

On peut effectivement retirer la négation ici, mais dans ce cas il faut la reporter dans le JSX.

Reply:
🔽 https://github.com/pass-culture/pass-culture-main/pull/13200/commits/77524184b3522d67208d2926a772d4737471a381